.riot/requirements/50d0377.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --no-annotate .riot/requirements/f20bd15.in
+#    pip-compile --no-annotate .riot/requirements/50d0377.in
```

Comment:
this is the only one that needed updating? 🤔 

there is also `min_compatible_versions.csv` and `lib-injection/min_compatible_versions.csv` which we might be able to remove it from as well?

Reply:
If I run the script `python ./scripts/min_compatible_versions.py`, other dependencies get updated in the file, is that OK?

Reply:
ah, yeah, makes sense!

as long as we run that and the "compile and prune lock files" one, then all good!